examples/package.json

```diff
@@ -33,13 +33,21 @@
       "@typescript-eslint/ban-ts-comment": 0,
       "@typescript-eslint/no-explicit-any": 0,
       "@typescript-eslint/explicit-module-boundary-types": 0,
+      "radix": 0,
```

Comment:
How come you have to disable this rule?

Reply:
The radix parameter is optional, defaults to base 10, typescript inserts it.  3 linting errors were present.

Project is being built to at least ES5 in all environments.  This "examples" is being built to es6.  This was introduced during the correction in behaviour of es5.

Reference: https://eslint.org/docs/latest/rules/radix
> On the other hand, if the code is targeting only ES5-compliant environments passing the radix 10 may be redundant. In such a case you might want to disallow using such a radix.

It's purely a question of style at this point.  At the time, it was valuable.  It's not enabled by "eslint:recommended".